lib/lhm/migrator.rb

```diff
@@ -32,14 +32,16 @@ def initialize(table, connection = nil)
     #   end
     #
     # @param [String] statement SQL alter statement
+    # @param [String] algorithm Algorithm that will be used in the DDL operation
     # @note
     #
     #   Don't write the table name directly into the statement. Use the #name
     #   getter instead, because the alter statement will be executed against a
     #   temporary table.
     #
-    def ddl(statement)
-      statements << statement
+    def ddl(statement, algorithm = nil)
```

Comment:
Keyword arguments help prevent breaking changes
```suggestion
    def ddl(statement, algorithm: nil)
```

Same for the rest of the file